config/application.rb

```diff
@@ -170,7 +170,8 @@ class Application < Rails::Application
       :app => {:enabled => true},
       :ldap => {:enabled => false},
       :permissions => {:enabled => false},
-      :sql => {:enabled => false}
+      :sql => {:enabled => false},
+      :audit => {:enabled => true}
```

Comment:
Nevermind, I figured this out. still trying to think if we could simplify this function somehow as it feels a bit over complicated to me.